src/st/st_update.h

```diff
@@ -29,7 +29,7 @@ s32 Random(void) {
     return (g_randomNext >> 0x18) & 0xFF;
 }
 
-#include <stage.h>
```

Comment:
This was seemingly only included to get the `extern u16 g_ItemIconSlots` so can be removed now